src/access.lua

```diff
@@ -137,14 +140,22 @@ local function build_payload_hash()
   return str.to_hex(payload_digest)
 end
 
---- Add the JWT header to the reqeust
+local function build_header_value(conf, jwt)
+  if (conf.include_credential_type == true) then
```

Comment:
@row1govender  I think here you can just do:

```
if conf.include_credential_type then
```

Reply:
Done